examples/nlp/nmt_tutorial_en_zh.py

```diff
@@ -0,0 +1,249 @@
+""" Copyright (c) 2019 NVIDIA Corporation
+See the tutorial and download the data here:
+https://nvidia.github.io/NeMo/nlp/
+neural-machine-translation.html#translation-with-pretrained-model
```

Comment:
But this link only contains EN-DE data and model. 
Are you aware of any public EN-Chinese dataset that can be linked there instead??

Reply:
"nmt_tutorial_en_zh.py" uses different tokenizers from "nmt_tutorial.py" and it contains some comments explaining why we need two separate tokenizers. It also discards weight sharing between the embedding layer and the output layer since they have different vocabulary sizes.  If we merge these two scripts to one we will make the script more 'generic'. For example, add some 'if...else...', add some parameters, add more comments to explain each case. Is that okay? What would you say? 

Merging "sentence_classification_with_bert_zh.py" and "ner_zh.py" is simpler by just adding some more comments for the parameters. 

About the data, I didn't find appropriate data for EN-ZH NMT and ZH NER task so I prepared some sample data format for users to refer to. They can prepare their data according to the sample format.

Reply:
nmt_tutorial.py has src_lang and tgt_lang arguments, so I think using if ... else to decide the tokenizer used is fine.